### PR TITLE
Trainium with neuron is not yet supported for Ubuntu 22.  Fix test_create expected message

### PR DESCRIPTION
**Pull request by dreambeyondorange**
### References
https://github.com/aws/aws-parallelcluster-cookbook/pull/2234

### Checklist
- [ ] Make sure you are pointing to **the right branch** and add a label in the PR title (i.e. **2.x** vs **3.x**)
- [ ] Check all commits' messages are clear, describing what and why vs how.
- [ ] Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- [ ] Check if documentation is impacted by this change.

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/integration-tests/tests/create/test_create.py [hunk 1]
```diff
@@ -207,8 +207,7 @@ def test_cluster_creation_with_invalid_ebs(
     expected_cfn_failure_reason = (
         "Failed to mount EBS volume. Please check /var/log/chef-client.log in the head "
         "node, or check the chef-client.log in CloudWatch logs. Please refer to "
-        "https://docs.aws.amazon.com/parallelcluster/latest/ug/troubleshooting-v3.html#"
-        "troubleshooting-v3-get-logs for more details on ParallelCluster logs."
```
Comment (hanwen-pcluste) [inline tests/integration-tests/tests/create/test_create.py:211]: Why this change?

Comment (dreambeyondorange) [reply tests/integration-tests/tests/create/test_create.py:211]: I didn't make the change.  I am just fixing the test to match this

Comment (hanwen-pcluste) [reply tests/integration-tests/tests/create/test_create.py:211]: ~~There isn't any changes regarding this message https://github.com/aws/aws-parallelcluster/blob/develop/cli/src/pcluster/resources/head_node/user_data.sh#L137~~

~~Are you sure this change is necessary?~~

**Review by hanwen-pcluste — APPROVED** [review]
